### DISCUSSION TITLE
Meetings

### DISCUSSION THREAD

**Discussion by Bhavesh-Kapur** [Bhavesh-Kapur/Car-Showroom-Management-System#3] [General]
Discussion about the project

**Comment by Bhavesh-Kapur** [comment]
today meet at 7pm

**Comment by Bhavesh-Kapur** [comment]
meeting link 
https://meet.google.com/ykn-qzhr-fgs

**Comment by PrinceV007** [comment]
guys sab join karlo its already 7:05, we will wait for 5 more minutes only

**Comment by Aman-sain** [comment]
join kr le accept to kro

**Comment by Bhavesh-Kapur** [comment]
todays meeting link
https://meet.google.com/iqa-sqzu-hgr
